include/envoy/config/grpc_mux.h

```diff
@@ -111,6 +113,13 @@ class GrpcMux {
    */
   virtual void resume(const std::string& type_url) PURE;
 
+  // For delta
+  virtual Watch* addOrUpdateWatch(const std::string& type_url, Watch* watch,
```

Comment:
nit: use string_view wherever possible

Reply:
This gets used as the key in a map, which does not seem to like string_views.